lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -70,17 +70,29 @@ def print_data(data)
 
     class CreateCommand < HammerCLIForeman::CreateCommand
 
+      VALIDATION_PER_PROIVIDER = {
+        'libvirt'   => [:option_url],
+        'ovirt'     => [:option_url, :option_user, :option_password, :option_datacenter],
+        'openstack' => [:option_url, :option_user, :option_password],
+        'racksapce' => [:option_url],
+        'ec2'       => [:option_region, :option_user, :option_password],
+        'vmware'    => [:option_user, :option_password, :option_datacenter, :option_server]
```

Comment:
@mbacovsky Thanks for reviewing my PR.

This is kind of vague but working solution to validate ComputeResource#create options. One thing to note in case of vmware[1] and ec2 URL option has been *smartly* bypassed with server and zone respectively.

[1] https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/vmware.rb#L423

I'll create a new patch to handle server side url validation(url should be validated based on provider)